spec/fortnox/api/repositories/invoice_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'spec_helper'
+require 'fortnox/api/repositories/invoice'
+require 'fortnox/api/models/invoice'
+
+describe Fortnox::API::Repository::Invoice do
+
+  before{
+    ENV['FORTNOX_API_BASE_URL'] = ''
+    ENV['FORTNOX_API_CLIENT_SECRET'] = ''
+    ENV['FORTNOX_API_ACCESS_TOKEN'] = ''
+  }
+
+  after{
+    ENV['FORTNOX_API_BASE_URL'] = nil
+    ENV['FORTNOX_API_CLIENT_SECRET'] = nil
+    ENV['FORTNOX_API_ACCESS_TOKEN'] = nil
+  }
+
+  let( :repository ){ Fortnox::API::Repository::Invoice.new }
+
+  describe '#save' do
+    context 'unsaved, new Invoice' do
```

Comment:
This context description is wrong, you are testing a saved invoice (or faking a saved invoice).